.github/dependabot.yml

```diff
@@ -0,0 +1,28 @@
+# Check for outdated packages
+# See:
+# https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/enabling-and-disabling-version-updates
+# And:
+# https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates
+
+version: 2
+
+updates:
+  - package-ecosystem: pip
+    # Where to search for requirements.txt, pipfile, pipfile.lock or
+    # setup.py* files. See here for supported requirement files:
+    # https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems
+    directory: ./
+    schedule:
+      interval: weekly
+    ignore:
+      - dependency-name: matplotlib
```

Comment:
I have no experience with dependabot but we'll see if weekly is too frequent or not.
Also, why ignore matplotlib updates?

Reply:
I also have no experience with that, but let's see.

Currently, we are more or less restricted to matplotlib version 3.2. We could go to 3.3, but than you will get deprecation warnings every time you import mdtools. That's somehow, because I set `plt.rc('text', usetex=True)` in [plot.py](https://github.com/andthum/mdtools/blob/66217f76915efd79bd84930b06e5ded1bbb11fa8/mdtools/plot.py#L35), which is a deprecated option in higher matplotlib versions. I have not looked into that issue further, because `plot.py` needs some bigger revisions anyway and it's probably not a good idea to render plots with TeX by default, because this doesn't work if you haven't TeX installed on you computer. But it gives you plots that are very consistent in style with your LaTeX documents.

Reply:
Ok, let's leave this for now.